src/search.c

```diff
@@ -1543,8 +1544,6 @@ static GString *get_grep_options(void)
 		g_string_append_c(gstr, 'i');
 	if (settings.fif_match_whole_word)
 		g_string_append_c(gstr, 'w');
-	if (settings.fif_recursive)
-		g_string_append_c(gstr, 'r');
```

Comment:
Yes, I know/remember the issue, I just meant to suggest it'd be better separated, and ask whether there was a particular reason to have it integrated.